frontend/src/pages/modelServing/screens/projects/NIMServiceModal/DeployNIMServiceModal.tsx

```diff
@@ -246,14 +285,14 @@ const DeployNIMServiceModal: React.FC<DeployNIMServiceModalProps> = ({
 
   return (
     <Modal
-      title="Deploy model with NVIDIA NIM"
+      title={`${editInfo ? 'Edit' : 'Deploy'} model with NVIDIA NIM`}
       description="Configure properties for deploying your model using an NVIDIA NIM."
       variant="medium"
       isOpen
       onClose={() => onBeforeClose(false)}
       footer={
         <DashboardModalFooter
-          submitLabel="Deploy"
+          submitLabel={editInfo ? 'Redeploy' : 'Deploy'}
```

Comment:
@vconzola did we ever decide on what we were calling the deploy button on edit Model Serving modals?

Reply:
@andrewballantyne I believe we're labeling the button "Redeploy". That's what I see in the product today.

Reply:
Perfect, thanks!